falkordb_gemini_kg/steps/graph_query_step.py

```diff
@@ -66,11 +50,17 @@ def run(self, question: str, retries: int = 5):
                         question=question, error=error
                     )
                 )
+                logger.debug(f"Cypher Prompt: {cypher_prompt}")
                 cypher_statement_response = self.chat_session.send_message(
                     cypher_prompt,
                 )
+                logger.debug(f"Cypher Statement Response: {cypher_statement_response}")
                 cypher = extract_cypher(cypher_statement_response.text)
                 logger.debug(f"Cypher: {cypher}")
+
+                if not cypher or len(cypher) == 0:
+                    return (None, None)
```

Comment:
does the orchestrator knows how to deal with these exceptions?

Reply:
It will get an empty response that he will probably pass forward. But there is no explicit instruction in that regard i believe